packages/suite/src/components/suite/asset-picker/components/AssetRow/AssetRowAccount/AccountAmount.tsx

```diff
@@ -0,0 +1,36 @@
+import { Account } from '@suite-common/wallet-types';
+import { asAmountSubunit, subunitsToUnits } from '@suite-common/wallet-utils';
+import { Column, Text } from '@trezor/components';
+import { BigNumber } from '@trezor/utils';
+
+import { BaseCurrencyValue } from 'src/components/suite/BaseCurrencyValue';
+import { FormattedCryptoAmount } from 'src/components/suite/FormattedCryptoAmount';
+
+interface AccountAmountProps {
+    account: Account;
+}
+
+export function AccountAmount({ account }: AccountAmountProps) {
```

Comment:
we use `const` on most of the places
```suggestion
export const AccountAmount({ account }: AccountAmountProps) {
```

Reply:
No. Context here https://github.com/trezor/trezor-suite/pull/22979#discussion_r2537939840